### DISCUSSION TITLE
Allow "SourceGame" to be anything.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#18195] [Feature requests] [answered]
For adding custom music or models from other games it could be a nice addition to have anything that isn't specifically listed as officially supported games (so that would display as custom) to read out the source game. This would still be listed under the custom tab, but it would display the correct game it originates from. There's quite a few custom things from other games like zoo tycoon, the sims, simcity, age of empires and more isometric games. As well as music from other coaster games, or other soundtracks.

**Comment by 733737** [comment]
this encourages copyright infringement so I don't think it will be added

**Reply by karst** [reply]
I don't see how leaving proper credit as custom unofficial content would encourage copyright infringement.

**Comment by 733737** [comment]
it's not proper credit, if the assets are being distributed illegally

**Reply by Lastorder-DC** [reply]
Garry's Mod also allows linking other source engine games, but we do not call that as "copyright infringement" since users have to buy game to link.
Also making two unrelated answers is also not helpfu~~n~~l at all.

If you have anything more to say, just comment below. do not add more "answer" to this post.

**Comment by Gymnasiast** [comment, marked as answer]
The field is there to categorise which of the RCT games content came from. Everything else is thus put under “custom”.
